### PR TITLE
Feature event logging

### PR DESCRIPTION
**Pull request by zmelnick**

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/event/DefaultEventDescriptionBuilder.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ public void buildPayloadTypes() {
      * @see gov.hhs.fha.nhinc.event.EventDescriptionBuilder#buildPayloadSize()
```
Comment (tparnell) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/event/DefaultEventDescriptionBuilder.java:82]: This @see is now incorrect. (check others as well)

File: Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/entity/EntityAdministrativeDistributionSecured.java [hunk 2]
```diff
@@ -47,6 +49,8 @@
     private EntityAdminDistributionOrchImpl orchImpl;
 
     @Override
+    @InboundMessageEvent(serviceType = "Admin Distribution", version = "1.0",
```
Comment (tparnell) [inline Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/entity/EntityAdministrativeDistributionSecured.java:53]: Is this the right event?  Shouldn't this differ from previous file?

File: Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/nhin/NhinAdministrativeDistribution.java [hunk 3]
```diff
@@ -49,6 +51,8 @@
     private NhinAdminDistributionOrchImpl orchImpl;
 
     @Override
+    @InboundMessageEvent(serviceType = "Admin Distribution", version = "1.0",
```
Comment (tparnell) [inline Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/nhin/NhinAdministrativeDistribution.java:55]: Same question - is InboundMessageEvent correct here?

File: Product/Production/Gateway/AdminDistribution_20/src/main/java/gov/hhs/fha/nhinc/admindistribution/_20/entity/EntityAdministrativeDistribution_g1.java [hunk 4]
```diff
@@ -45,6 +47,8 @@
     private EntityAdminDistributionOrchImpl orchImpl;
 
     @Override
+    @InboundMessageEvent(serviceType = "Admin Distribution", version = "2.0",
```
Comment (tparnell) [inline Product/Production/Gateway/AdminDistribution_20/src/main/java/gov/hhs/fha/nhinc/admindistribution/_20/entity/EntityAdministrativeDistribution_g1.java:50]: Same comment re:correct type.

File: Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/entity/EntityAdministrativeDistributionSecured.java (hunk 2 above)
Comment (zmelnick) [inline Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/entity/EntityAdministrativeDistributionSecured.java:53]: No. One is the secured event, the other is the unsecured version of the same event.

File: Product/Production/Services/AdminDistributionCore/src/main/java/gov/hhs/fha/nhinc/admindistribution/aspect/InboundProcessingDescriptionBuilder.java [hunk 5]
```diff
@@ -0,0 +1,175 @@
```
Comment (tparnell) [inline Product/Production/Services/AdminDistributionCore/src/main/java/gov/hhs/fha/nhinc/admindistribution/aspect/InboundProcessingDescriptionBuilder.java]: Are there unit tests for this file?

File: Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/entity/EntityAdministrativeDistributionSecured.java (hunk 2 above)
Comment (tparnell) [inline Product/Production/Gateway/AdminDistribution_10/src/main/java/gov/hhs/fha/nhinc/admindistribution/_10/entity/EntityAdministrativeDistributionSecured.java:53]: Ack.  Thanks for double checking.

**Pull request merged**